ansible/roles/common/tasks/install-packages.yml

```diff
@@ -11,6 +11,7 @@
     - tree
     - gcc
     - python-devel
+    - yum-versionlock
```

Comment:
According to [0], the package name is `yum-plugin-versionlock`. I'm guessing they keep `yum-versionlock` as an alias for backwards compatibility?


[0] https://access.redhat.com/solutions/98873

Reply:
Good catch! Fixing it.